boot/makebootable_test.go

```diff
@@ -434,6 +452,80 @@ grade=dangerous
 `)
 }
 
+func (s *makeBootable20Suite) TestMakeBootable20RunModeInstallBootConfigErr(c *C) {
+	dirs.SetRootDir("")
```

Comment:
Nitpick, this could probably go to SetUpTest (or be omitted), it seems that most of the existing makeBootable20UbootSuite tests start with this.

Reply:
Done